ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -206,6 +206,22 @@
     reason="Test runs ONLY on OSD or ROSA cluster",
 )
 
+ms_provider_required = pytest.mark.skipif(
+    not (
+        config.ENV_DATA["platform"].lower() in MANAGED_SERVICE_PLATFORMS
+        and config.ENV_DATA["cluster_type"].lower() == "provider"
+    ),
+    reason="Test runs ONLY on managed service provider cluster",
+)
+
+ms_consumer_required = pytest.mark.skipif(
+    not (
+        config.ENV_DATA["platform"].lower() in MANAGED_SERVICE_PLATFORMS
+        and config.ENV_DATA["cluster_type"].lower() == "consumer"
```

Comment:
```suggestion
        and config.ENV_DATA.get("cluster_type", "").lower() == "consumer"
```
"platform" key should be part of all configs but "cluster_type" might not be set (not managed platforms or converged mode). This can result in a key error.

Reply:
I think not. Define this to default config will take care of this.  https://github.com/red-hat-storage/ocs-ci/blob/8f806f709e8190d545ddb924723364f4c9c9d031/ocs_ci/framework/conf/default_config.yaml#L198

Reply:
Ok, in that case it could work. I just recommend to run TestAddNode test case also at not managed service platform as a verification job to be sure.